infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -71,16 +71,60 @@ type FeatureStoreServices struct {
 // OfflineStore configures the deployed offline store service
 type OfflineStore struct {
 	ServiceConfigs `json:",inline"`
+	// +optional
+	Persistence *OfflineStorePersistence `json:"persistence,omitempty"`
+}
+
+// OfflineStorePersistence configures the persistence settings for the offline store service
+type OfflineStorePersistence struct {
+	// +optional
+	FilePersistence *OfflineStoreFilePersistence `json:"file,omitempty"`
+}
+
+// OfflineStorePersistence configures the file-based persistence for the offline store service
+type OfflineStoreFilePersistence struct {
+	// +optional
+	// +default:value=dask
```

Comment:
we don't want to apply defaults here as they will appear in the `spec`. the application of defaults should occur w/ controller logic to `status.applied`.